src/components/controls/WalletIntegration.jsx

```diff
@@ -58,6 +50,15 @@ function WalletIntegration({
       // Set Modal
       showModal('Loading user data . . .')
 
+      // Gnosis Safe Fix
+      Promise.race([
+        delay(500),
+        safeInjected,
```

Comment:
If you want there's a proper `safeInjected` implementation in https://github.com/gnosis/dx-mgn-pool-react/blob/99e490c5e2a6d878b8db4303c62f059b2ecd65c4/src/api/providers/index.js#L78 that waits for an event from Safe on `window`
But delay works too.

Reply:
idk this was your code i moved :man_shrugging:

Reply:
will take a look at this tho thanks